inst/@sym/solve.m

```diff
@@ -31,9 +31,9 @@
 %% @example
 %% @group
 %% syms x
-%% solve(x == 2*x + 6, x)
+%% solve (x == 2 * x + 6, x)
```

Comment:
okis, the code its updated.
i'm asking because you mentioned you want follow some code style, and your old code don't follow it..
Thx.